libres/lib/include/ert/enkf/row_scaling.hpp

```diff
@@ -1,67 +1,21 @@
-/*
-  Copyright (C) 2020  Equinor ASA, Norway.
-
-  The file 'row_scaling.hpp' is part of ERT - Ensemble based Reservoir Tool.
-
-  ERT is free software: you can redistribute it and/or modify
-  it under the terms of the GNU General Public License as published by
-  the Free Software Foundation, either version 3 of the License, or
-  (at your option) any later version.
-
-  ERT is distributed in the hope that it will be useful, but WITHOUT ANY
-  WARRANTY; without even the implied warranty of MERCHANTABILITY or
-  FITNESS FOR A PARTICULAR PURPOSE.
-
-  See the GNU General Public License at <http://www.gnu.org/licenses/gpl.html>
-  for more details.
-*/
+#pragma once
+#include <ert/res_util/matrix.hpp>
+#include <vector>
+#include <memory>
 
-#ifndef ROW_SCALING_H
-#define ROW_SCALING_H
+class RowScaling : public std::enable_shared_from_this<RowScaling> {
+    size_t m_resolution = 1000;
```

Comment:
Do we know what `m_resolution` is and why it's 1000?

Reply:
@joakim-hove